### DISCUSSION TITLE
Issue while loading certain collections

### DISCUSSION THREAD

**Discussion by geekychandraul** [milvus-io/milvus#28842] [Q&A and General discussion]
Hi Team,
I am facing a weird issue.
My all other collections are getting loaded but there are two collection which are not loading. when I try to load them its takes forever to load but the process never finished. unfortunalty this issue is occurring only on production.
we are milvus 2.2.6 and these collections have approx 60 partitions in them. one collection may have approx 200 data and other one has approx 5000 records.
Any help is appreciated.

**Comment by yhmo** [comment]
Which version of the SDK you are using? There are some minor changes between 2.2.6 and 2.2.9 for load()/get_load_state() interfaces. Incompatible sdk might be stuck during loading.

**Reply by xiaofan-luan** [reply]
highly recommened to upgrade to 2.2.15 and related pymilvus
